icu4c/readme.html

```diff
@@ -3,7 +3,7 @@
 
 <html lang="en-US" xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-US">
   <head>
-    <title>ReadMe for ICU 68.2</title>
```

Comment:
One thought and/or thing to call out: 
FWIW, I think the changes to this file are part of the "Update readme" task(s), so FYI for @echeran.
Though this PR is only changing the ICU4C readme -- so the ICU4J readme would still need to be updated.

Reply:
I think the version update on this file are fine and don't overlap with the remaining work on the "Update readme" task. The [ICU4C Readme](https://unicode-org.github.io/icu/userguide/icu4c-readme.html) and [ICU4J Readme](https://unicode-org.github.io/icu/userguide/icu4j-readme.html) are technically in the user guide. They exist in a version-agnostic way, and they correspond to code releases automatically because they are in git. The remaining work is to de-duplicate info and/or reorganize ICU4C build instructions and the separate build tips file/pages. The change here looks good, and thanks for the heads up!

Reply:
Thanks @echeran !

> I think the version update on this file are fine and don't overlap with the remaining work on the "Update readme" task

Hm, in that case, I think we'd want/need to update the version numbers in the ICU4J readme file as part of this change as well. (It still says version 68 for example: https://github.com/unicode-org/icu/blob/master/icu4j/readme.html#L17 )

Reply:
@jefgen Thanks, yes, you're right. I meant to affirm the need to change the version numbers here, but I didn't say it clearly.